contrib/test/integration/e2e.yml

```diff
@@ -51,7 +51,18 @@
 - name: disable SELinux
   command: setenforce 0
 
-- name: run e2e tests
-  shell: "{{ e2e_shell_cmd | regex_replace('\\s+', ' ') }}"
-  args:
-    chdir: "{{ ansible_env.GOPATH }}/src/k8s.io/kubernetes"
+- block:
+
+    - name: Disable swap during e2e tests
+      command: 'swapoff -a'
+      when: not e2e_swap_enabled
```

Comment:
This bothers me more than I think it should. It creates an unpleasant condition: if the playbook dies between here and the `swapon`, a subsequent playbook will rerun the entire make-swapfile block. Can you explain why you wish to use a tmpfile as swapfile instead of a constant path? With a constant path, `swap.yml` would be much simpler and would avoid having to use `block`.

Reply:
That's what the ``always`` part of the ``block`` mitigates. If the playbook dies, ``swapon -a`` gets run.  Just about the only case where that won't happen is if there are networking issues, but then that's orthogonal to the playbook puking.

Reply:
Quick comment: I understand `always`; my concern is that there's a long, long time between the `swapoff` and the `always` cleanup. That's a large window for Murphy: network disconnect, system crash, accidental ^C, anything that will prevent ansible from finishing. I'm just neurotic that way.

Reply:
All true, and I get a bit similarly neurotic.  However, anything Murphy can break in that long duration, isn't anything that wouldn't also break the test w/o this patch.  So all outcomes are still: Throw away the VM and try again.  I think I would go [more] insane trying to chase down all the ways CI could break.  Sometimes it's easier to just accept that once in a while, a human vision-orbs and squishy-computer are required.